components/jupyter-web-app/frontend/src/app/uis/rok/rok-resource-form/rok-jupyter-lab-selector/rok-jupyter-lab-selector.component.ts

```diff
@@ -47,7 +47,7 @@ export class RokJupyterLabSelectorComponent implements OnInit {
   }
 
   setLabValues(lab: JupyterLab) {
-    this.parentForm.get("customImage").setValue(lab.image);
+    this.parentForm.get("customImage").setValue(lab.image.trim());
```

Comment:
I think you meant to use the `.strip()` function here right?

Reply:
I don't think [so](https://www.w3schools.com/jsref/jsref_trim_string.asp), but I may be wrong.

Reply:
@ianhellstrom my bad, I got confused for a second and thought this was also python code